packages/generator-common/src/options-parser.spec.ts

```diff
@@ -92,6 +107,45 @@ describe('options parser', () => {
       });
     });
 
+    it('includes using glob using cwd', () => {
+      expect(parseOptions({ include }, { include: '*.json' }).include).toEqual(
+        absoluteJsonPaths
+      );
+    });
+
+    it('includes using glob using root', () => {
```

Comment:
This test didn't work on windows. The one above works, so I assume it's because of `resolve` in the pattern, not sure.

Reply:
I tried to split it - can you try?